packages/react-native/README.md

```diff
@@ -87,7 +87,65 @@ error('Hello, World!');
 
 ```
 
-## Contributing
+### Network Integration
 
-See the [contributing guide](CONTRIBUTING.md) to learn how to contribute to the repository and the development workflow.
+The automatic capture of network requests can be achieved in a few different ways depending on the platform and the networking library being used.
 
+#### iOS
+
+When initializing via JS, pass `enableNetworkInstrumentation: true` as one of the options to `init`:
+
+```javascript
+import { init, SessionStrategy } from '@bitdrift/react-native';
+
+init('<api key>', sessionStrategy: SessionStrategy.Activity, {
+  enableNetworkInstrumentation: true
+});
+```
+
+Enabling this via the Objective-C API is not yet supported if initialization is done via the native API - please reach out if there is interest in this feature.
+
+
+#### Android
+
+To enable network integration in Android a Gradle plugin needs to be added to the project. This can be done by adding a dependency on the `io.bitdrift.capture-plugin` plugin in the `plugins` block of the apps's `build.gradle` file:
+
+```gradle
+plugins {
+    id 'io.bitdrift.capture-plugin' version '0.16.6'
```

Comment:
I would just something like `id 'io.bitdrift.capture-plugin' version '<version>'` to avoid having to update this number on every release

Reply:
Yeah good idea. It's a bit awkward since this version is configured independently to the version of the SDK used by the RN package but I'll just add a note about this